README.md

```diff
@@ -167,6 +167,7 @@ To run style check manually, simply run the following from the top level folder.
 ```
 pip install -r ci/checks/python-style-requirements.txt
 flake8 pyclaragenomics/
+flake8 pyclaragenomics/ --config pyclaragenomics/.flake8.cython  # for Cython style check
```

Comment:
can you also update the README to say `source pyclaragenomics/style_check`?

Reply:
Changed this to be exected in a new shell, and not sourcing